src/components/TokenDetails.tsx

```diff
@@ -103,7 +109,8 @@ const TokenDetails = ({ id }: Props) => {
                   padding: [0, 0, "default"]
                 }}
                 onClick={() => {
-                  window.open(`${MEDIA_URL}/${token.tokenId}.png`)
+                  const mediaUrl = contractConfig[0].MEDIA_URL
```

Comment:
ditto the above

Reply:
Actually this one definitely shouldn't be hardcoded right? since media urls will be different?

Reply:
media_url is this context I believe might be referencing our media proxy? However, you should be able to get the media_url from the tokens_metadata. `token.media_url`

Reply:
Address with [071be03](https://github.com/ArtBlocks/artblocks-engine-react/pull/10/commits/071be0363e6a4237b3733c943f00446b578e7f0c)

@grantoesterling yes the media url will be different for each contract - at least that has been my experience with plottables.

@Jbern16 I'm not seeing the media_url in the tokens_metadata but maybe I am missing something. Is this coming from the subgraph somewhere?

Reply:
ah - my apologies, this is from our off chain api. We should be opening that up to the public soon, but perhaps until then we stick to the Subgraph here